net/nlbwmon/Makefile

```diff
@@ -1,13 +1,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=nlbwmon
-PKG_RELEASE:=2
+PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://github.com/jow-/nlbwmon.git
-PKG_SOURCE_DATE:=2019-06-06
-PKG_SOURCE_VERSION:=4574e6e80e3a5ab8de65886baa0d563dfee589ff
-PKG_MIRROR_HASH:=559a8b04e8942c83e31a6a5e2576a5308fdc16349ed29e0b2d9fc182d715976d
+PKG_SOURCE_DATE:=2019-06-13
+PKG_SOURCE_VERSION:=e921ca0af9957d3cb05797acfb8bde4d7d2278e5
+PKG_MIRROR_HASH:=1f6ed5086e978beeba4c22c255b4ed893d598119c3f2c03061501464910ea421
 
 CMAKE_INSTALL:=1
```

Comment:
This should be removed if nlbwmon doesn't need to install anything to InstallDev. PKG_INSTALL is included with cmake.mk: https://github.com/openwrt/openwrt/blob/master/include/cmake.mk#L3

Reply:
PKG_BUILD_PARALLEL:=1 should be added as well.